valgrind_output_compactor.awk

```diff
@@ -0,0 +1,59 @@
+# Downloaded from https://wiki.wxwidgets.org/Valgrind_Suppression_File_Howto on 2019/10/24
+
+#! /usr/bin/awk -f
```

Comment:
The shebang line should be first in script files (that's how `ld` knows whom to pass it to), so move the "downloaded" comment from above to be under this line :)